lib/ansible/plugins/lookup/random_choice.py

```diff
@@ -36,4 +38,11 @@ class LookupModule(LookupBase):
 
     def run(self, terms, inject=None, **kwargs):
 
-        return [random.choice(terms)]
+        ret = terms
+        if terms:
+            try:
+                ret = [random.choice(terms)]
+            except Exception as e:
+                raise AnsibleError("Unable to choose random term: %s" % to_native(e))
```

Comment:
The error message could give more constructive feedback:
`random requires a non-empty list of elements.`